web_customisable_date_format/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
should be png, but it would be better to switch to new readme format https://github.com/OCA/maintainer-tools/tree/master/template/module/readme :wink: